apps/swirl-docs/src/components/Documentation/EndpointDescription.tsx

```diff
@@ -121,6 +127,30 @@ export const EndpointDescription: FunctionComponent<EndpointDescription> = ({
             </div>
           </div>
         )}
+
+        {!!endpoint.globalErrorCodes?.length && (
```

Comment:
thought (non-blocking): could be put in a component

Reply:
I would prefer to split this large component in several sub-components outside this PR

Reply:
sure. can we make a ticket for that?

Reply:
APP-20517